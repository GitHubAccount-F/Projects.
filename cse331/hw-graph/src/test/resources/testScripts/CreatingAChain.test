#Tests to see if we can create a chain of nodes.
#For example, A-->B-->C-->A

#Creates an empty graph
CreateGraph graph1

#Adds nodes
AddNode graph1 one
AddNode graph1 two
AddNode graph1 three

#Creates edges
AddEdge graph1 one two e1
AddEdge graph1 two three e2
AddEdge graph1 three one e1

#Lists the nodes
ListNodes graph1

#Lists the child nodes, where we should
#be able to tell that their is a chain present
ListChildren graph1 one
ListChildren graph1 two
ListChildren graph1 three