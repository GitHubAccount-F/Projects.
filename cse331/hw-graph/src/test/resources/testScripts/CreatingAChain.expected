#Tests to see if we can create a chain of nodes.
#For example, A-->B-->C-->A

#Creates an empty graph
created graph graph1

#Adds nodes
added node one to graph1
added node two to graph1
added node three to graph1

#Creates edges
added edge e1 from one to two in graph1
added edge e2 from two to three in graph1
added edge e1 from three to one in graph1

#Lists the nodes
graph1 contains: one three two

#Lists the child nodes, where we should
#be able to tell that their is a chain present
the children of one in graph1 are: two(e1)
the children of two in graph1 are: three(e2)
the children of three in graph1 are: one(e1)